lib/web_ui/lib/src/ui/window.dart

```diff
@@ -73,18 +73,14 @@ enum AppLifecycleState {
   ///
   /// When the application is in this state, the engine will not call the
   /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
-  ///
-  /// Android apps in this state should assume that they may enter the
-  /// [suspending] state at any time.
   paused,
 
-  /// The application will be suspended momentarily.
-  ///
-  /// When the application is in this state, the engine will not call the
-  /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
+  /// The application is detached from view.
   ///
-  /// On iOS, this state is currently unused.
-  suspending,
+  /// When the application is in this state, the engine is running without
+  /// a view. It can either in the progress of creating a view, or the view
```

Comment:
In the process of creating a view sounds a bit vague and/or overly inclusive. One might assume this would happen with every view creation process of any normal full-flutter apps, which isn't the case.

Reply:
It depends on the dart code the the flutter tree might or might not exist. I will just mention the engine is running without a platform UI